stop.sh

```
@@ -0,0 +1,64 @@
+
+# Функция для остановки сервиса
+stop_service() {
+    local service_name=$1
+    local script_name=$2
+    local pid_file="logs/${service_name}.pid"
+    
+    print_colored "🛑 Stopping ${service_name}..." "$YELLOW"
+    
+    # Проверяем PID файл
+    if [ -f "$pid_file" ]; then
+        local pid=$(cat "$pid_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)